src/elisp/treemacs-extensions.el

```diff
@@ -447,28 +449,33 @@ additional keys."
                          ,(format "The handle to update the the %s extension." name))
                        (defun ,ext-name ()
                          (treemacs-with-writable-buffer
-                          (-let [pr (make-treemacs-project
+                          (let ((pr (make-treemacs-project
                                      :name ,root-label
                                      :path ,root-key-form
-                                     :path-status 'extension)]
+                                     :path-status 'extension))
+                                (button-start (point)))
                             (treemacs--set-project-position ,root-key-form (point-marker))
                             (setq-local ,project-var-name pr)
-                            (insert (propertize "a"
+                            (insert (propertize "\n"
                                                 'button '(t)
                                                 'category 'default-button
+                                                'invisible t
+                                                'cursor-sensor-functions (list (lambda (_ previous-position change)
```

Comment:
`invisible` should not be selectable, but for some reason that's not the case. Use `cursor-sensor-mode` to skip to the next button on enter.